NativePrograms/PokemonSV/Programs/EggFetcher_Core.c

```diff
@@ -47,6 +47,8 @@ int main(void) {
     };
     pbf_press_button(BUTTON_A, 5, 500);
 
+    fly_to_gate();

```

Comment:
Sorry I just got started with reviewing. Previously fly_to_gate() had a high fail rate if the player is already at or close to the fly spot as the cursor would snap to the player icon instead of the fly spot. Could you stress test this and if there are fail cases add a move forward sequence (using map marker alignment) so the player is decently away from the fly spot? (this would be useful to implement after the hatching loop as well)

Reply:
I have been using this same code, non stop for the past month or so and till now there has been no instance where this failed. It works no matter where you are standing in the general area.